parsers/nmparser/get_model_output.go

```diff
@@ -12,7 +12,7 @@ import (
 // GetModelOutput populates and returns a ModelOutput object by parsing files
 // ParameterData is parsed from the ext file when useExtFile is true
 // ParameterData is parsed from the lst file when useExtFile is false
-func GetModelOutput(filePath string, verbose bool, useExtFile bool) ModelOutput {
+func GetModelOutput(filePath string, verbose bool) ModelOutput {
```

Comment:
why did you remove the ability to use the ext file or not. What if the user knows explicitly they don't have it or wants to suppress using it. Attempting to read and blowing up is not a suitable solution

Reply:
I thought you were onboard with "if the files exists, use it".  BTW, the flag was always true in the last implementation.

Reply:
yes, as a default! not an only way. there is no reason for us not to expose turning it off if needed. What if its there but can't be parsed correctly, for example with #31 what if they want to compare the summary from the lst vs the ext file as they suspect a bug but aren't sure, etc.

Reply:
Ok, so do we want a flag for grd as well?

Reply:
yep - and any other file we pull in that we could also get those values from the lst file, if we need to.

I see the lst file only parsing as a fallback that we will need to monitor in the future based on user need. My hope is that we can actually move the parsing out to all these much more consistent files, and parse as little as possible from the lst.